tests/test_utils.py

```diff
@@ -15,11 +15,15 @@
 
 def test_nepers2db():
     assert abs(neper2db(1.5) - 8.186258123051049e+05) < 1e-6, "Point check of nepers2db incorrect"
+    assert np.max(np.abs(neper2db(1.5 * np.ones((10,10))) - 8.186258123051049e+05 * np.ones((10,10)))) < 1e-6, \
+        "Point check of nepers2db incorrect"
     return
 
 
 def test_db2nepers():
-    assert abs(db2neper(1.6) - 2.931742395517710e-06) < 1e-6, "Point check of nepers2db incorrect"
+    assert abs(db2neper(1.6) - 2.931742395517710e-06) < 1e-6, "Point check of db2nepers incorrect"
+    assert np.max(np.abs(db2neper(1.6 * np.ones((10,10))) - 2.931742395517710e-06 * np.ones((10,10)))) < 1e-6, \
```

Comment:
Maybe define `2.931742395517710e-06` and `np.ones((10,10))` as variables? It can help with readability?